test/e2e-test/pages/helper-functions.ts

```diff
@@ -38,10 +39,11 @@ async function createUser() {
 }
 
 async function signInHelper() {
-  if (process.env.NODE_ENV !== 'development') {
+  const environment: string = process.env.NODE_ENV;
+  if (environment !== 'development') {
```

Comment:
not that is wrong, but let's try not to refactor things that are working or not needed? mostly if the PR already includes 86 files to review lol

Reply:
this is the same logic that was there before I've only just cached the variable into a cons, but I can revert the change.